doc/specs/#2046 - Command Palette.md

```diff
@@ -248,6 +288,63 @@ For example, consider the following list of commands:
 As the user types, we should **bold** each matching character in the command
 name, to show how their input correlates to the results on screen.
 
+### Commandline Mode

```

Comment:
Minor questions...
- if a user includes "wt.exe" in the input string, can we _not_ have a parsing error?
- should we add a `new-window` command? It'd be redundant to just running wt.exe in a terminal window, but I feel like it creates a better story for "use commandline mode to run wt.exe commands".

Reply:
* Uh, yea? I guess we could, though I thought it more ergonomic to just do `>new-tab`
* There's already #1051